### PR TITLE
DOC: fix RT03 for pandas.core.groupby.DataFrameGroupBy.hist

### PR DESCRIPTION
**Pull request by tuhinsharma121**
fixes

```
pandas.core.groupby.DataFrameGroupBy.hist RT03
```

### CODE REVIEW

File: pandas/core/groupby/generic.py [hunk 1]
```diff
@@ -2694,7 +2694,9 @@ def hist(
 
         Returns
         -------
-        matplotlib.Axes or numpy.ndarray of them
+        matplotlib.Axes or numpy.ndarray
+            A `matplotlib.Axes` object or an array of `Axes` objects, depending on
```
Comment (rhshadrach) [inline pandas/core/groupby/generic.py:2698]: Sphinx is used to build the documentation, so you need double backticks. E.g. ` ``matplotlib.Axes`` `

**Review by rhshadrach — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: pandas/core/groupby/generic.py (hunk 1 above)
Comment (tuhinsharma121) [reply pandas/core/groupby/generic.py:2698]: got it. changing it.

Comment (tuhinsharma121) [reply pandas/core/groupby/generic.py]: done 👍

**Review by rhshadrach — APPROVED** [review]
lgtm

**Comment by rhshadrach** [comment]
Thanks @tuhinsharma121!

**Pull request merged**